lib/workers/repository/update/pr/index.ts

```diff
@@ -364,7 +369,14 @@ export async function ensurePr(
         logger.info({ pr: existingPr.number, prTitle }, `PR updated`);
         setPrCache(branchName, prFingerprint, true);
       }
-      return { type: 'with-pr', pr: existingPr };
+      return {
+        type: 'with-pr',
+        pr: {
+          ...existingPr,
+          bodyStruct: getPrBodyStruct(prBody),
+          title: prTitle,
```

Comment:
are you sure you want to do these even if dryRun?

Reply:
Shouldn't matter I think

We don't use the returned info except `pr.number`
And the `prBody` & `prTitle` are generated anyways so better use correct the data